src/App.tsx

```diff
@@ -1,61 +1,125 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
 export const App = () => {
+  const getUserById = (id: number, errorMessage = 'User not found'): User => {
+    const user = usersFromServer.find(usr => usr.id === id);
+
+    if (!user) {
+      throw new Error(errorMessage);
+    }
+
+    return user;
+  };
```

Comment:
The `getUserById` function throws an error if a user is not found. This could potentially lead to unhandled exceptions if the function is called with an invalid user ID. Consider handling this scenario more gracefully, perhaps by returning a default user object or `null` and handling the absence of a user in the calling code.